src/vcpkg/base/files.cpp

```diff
@@ -2358,7 +2358,7 @@ namespace vcpkg
         virtual std::vector<Path> get_files_recursive(const Path& dir, std::error_code& ec) const override
         {
             std::vector<Path> result;
-            Path out_base = dir;
+            const Path& out_base = dir;
             get_files_recursive_impl(result, dir, out_base, ec, true, true, true);
```

Comment:
This seems wrong: Either the caller explicitly needed this to be a copy to call a non-`const&` overload of `get_files_recursive_impl` or similar, or it should just be:

```suggestion
            get_files_recursive_impl(result, dir, dir, ec, true, true, true);
```

Ditto below.

Reply:
```
        static void get_files_recursive_impl(std::vector<Path>& result,
                                             const Path& base,
                                             const Path& out_base,
                                             std::error_code& ec,
                                             bool want_directories,
                                             bool want_regular_files,
                                             bool want_other)
```

Pretty sure it's the latter...

Reply:
Other occurrences aren't fixed.